src/plugins/data_table/dataTables.colReorder.js

```diff
@@ -920,6 +933,165 @@
       this.s.dt.oInstance.oApi._fnSaveState(this.s.dt);
     },
 
+    /**
+     * Initialize sticky table header
+     *  @method  _fnSetupStickyTableHeader
+     *  @param   Int offsetTop number of pixels from top to stick header
+     *  @returns void
+     *  @private
+     */
+     "_fnSetupStickyTableHeader" : function(offsetTop) {
+       if (!offsetTop) { return } else if (offsetTop === true) {
+         offsetTop = 0;
+       }
+
+       // User must trigger 'tableLoaded' event after DataTable has been appended to DOM
+       // We need this to calculate offsets correctly
+       $(window).off('tableLoaded', start.bind(this));
+       $(window).on('tableLoaded', start.bind(this));
+
+       function start() {
+         var th, minWidth, minX;
+         var $window           = $(window);
+         var navbarHeight      = offsetTop;
```

Comment:
should name this more generically since it's in the library code (and actually why not keep `offsetTop` .. no reason to map to another var name)